### DISCUSSION TITLE
Where is the list of location?

### DISCUSSION THREAD

**Discussion by chrisza-tw** [Green-Software-Foundation/carbon-aware-sdk#133] [General]
I tried Carbon Aware SDK CLI and it's require parameter location. However, I don't know what are available locations I can use. 

I currently use "westus" as sample documentation but I wonder if there is a list of locations exists somewhere I can look up to?

**Comment by Willmish** [comment]
Hi @chrisza-tw , currently the locations are based on the Azure regions - target is to have any cloud provider names being directly mapped to data provider regions. Here is what mapping is currently being used: https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/src/CarbonAware.LocationSources/CarbonAware.LocationSources.Azure/src/azure-regions.json

It is worth noting, not all data providers have data for all locations. Currently SDK supports [WattTIme](https://www.watttime.org/) and local data (electricityMaps is under development here: https://github.com/Green-Software-Foundation/carbon-aware-sdk/tree/feat/electricity-map
